CHANGES.md

```diff
@@ -20,6 +20,7 @@ Change Log
 * Fixed removing multiple event listeners within event callbacks. [#5827](https://github.com/AnalyticalGraphicsInc/cesium/issues/5827)
 * Running `buildApps` now creates a built version of Sandcastle which uses the built version of Cesium for better performance.
 * Fixed a tileset traversal bug when the `skipLevelOfDetail` optimization is off. [#5869](https://github.com/AnalyticalGraphicsInc/cesium/issues/5869)
+* Adds `invertClassification` and `invertClassificationColor` to `Scene`. When `invertClassification` is `true`, any 3D Tiles geometry that is not classified will have its color multiplied by `invertClassificationColor`. [#5836](https://github.com/AnalyticalGraphicsInc/cesium/pull/5836)
```

Comment:
Does this also apply to the classification primitives?  I think so, right?  If so, then update this.

Otherwise, if this is 3D Tiles specific, maybe rename to `invert3DTilesClassification` and `invert3DTilesClassificationColor`?

Longer-term, would this move to per-primitive/tileset properties that could use the bounding volume as a scissor?  Maybe an event could notify the scene when these are enabled so that the main rendering pipeline still has the information it needs.

Reply:
Updated. It applies to `ClassificationPrimitive` and `GroundPrimitive`.

Yes, longer-term we could move this to per-primitive/tileset properties.